sdk/storage/inc/common/memory_stream.hpp

```diff
@@ -5,13 +5,16 @@
 
 #include "http/stream.hpp"
 
+#include <memory>
 #include <vector>
 
 namespace Azure { namespace Storage {
 
   class MemoryStream : public Azure::Core::Http::BodyStream {
   public:
-    explicit MemoryStream(const uint8_t* data, std::size_t length) : m_data(data), m_length(length) {}
+    explicit MemoryStream(const uint8_t* data, std::size_t length) : m_data(data), m_length(length)
```

Comment:
@JinmingHu-MSFT , is there a reason to have this memoryStream and the one from Azure::Core ?
I would say let's have just one

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/215

Reply:
replied in that issue.